spring-cloud-aws-sqs/src/test/java/io/awspring/cloud/sqs/SqsIntegrationTests.java

```diff
@@ -0,0 +1,522 @@
+
+import static io.awspring.cloud.sqs.config.SqsFactoryOptions.withOptions;
+import static java.util.Collections.singletonMap;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.testcontainers.containers.localstack.LocalStackContainer.Service.SQS;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.awspring.cloud.messaging.support.config.MessageListenerContainerFactory;
+import io.awspring.cloud.messaging.support.listener.AsyncErrorHandler;
+import io.awspring.cloud.messaging.support.listener.AsyncMessageInterceptor;
+import io.awspring.cloud.messaging.support.listener.AsyncMessageListener;
+import io.awspring.cloud.messaging.support.listener.MessageHeaders;
+import io.awspring.cloud.messaging.support.listener.MessageListenerContainerRegistry;
+import io.awspring.cloud.messaging.support.listener.acknowledgement.AsyncAckHandler;
+import io.awspring.cloud.messaging.support.listener.acknowledgement.AsyncAcknowledgement;
+import io.awspring.cloud.sqs.annotation.EnableSqs;
+import io.awspring.cloud.sqs.annotation.SqsListener;
+import io.awspring.cloud.sqs.config.SqsConfigUtils;
+import io.awspring.cloud.sqs.config.SqsMessageListenerContainerFactory;
+import io.awspring.cloud.sqs.listener.AsyncMessageHandlerMessageListener;
+import io.awspring.cloud.sqs.listener.SqsMessageHeaders;
+import io.awspring.cloud.sqs.listener.Visibility;
+import java.util.Objects;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.stream.IntStream;
+import org.jetbrains.annotations.Nullable;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.messaging.MessageHandler;
+import org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.util.Assert;
+import org.springframework.util.StopWatch;
+import software.amazon.awssdk.services.sqs.SqsAsyncClient;
+import software.amazon.awssdk.services.sqs.model.QueueAttributeName;
+import software.amazon.awssdk.services.sqs.model.ReceiveMessageResponse;
+import software.amazon.awssdk.services.sqs.model.SendMessageBatchRequestEntry;
+
+/**
+ * @author Tomaz Fernandes
+ * @since 3.0
+ */
+@SpringBootTest
+@DirtiesContext
+@TestPropertySource(properties = { "cloud.aws.credentials.access-key=noop", "cloud.aws.credentials.secret-key=noop",
```

Comment:
Note that properties have changed to `spring.cloud.aws` prefix. Hopefully you can follow docs: https://docs.awspring.io/spring-cloud-aws/docs/3.0.0-SNAPSHOT/reference/html/index.html